### PR TITLE
65 create readme

### PR DESCRIPTION
**Pull request by DesmondAldridge**
Made the README. We'll still need to add gifs of the app, and whatever changes/cleanups you all would like.

### CODE REVIEW

**Review by JesseJ713 — COMMENTED** [review]
I see this includes the commit history of the CSS styling you just requested. The README looks good, but I can't approve this until the consensus for the CSS has been approved. We'll be able to merge this once CSS has been given the green light. Solid README!

**Review by tylerdrewwork — APPROVED** [review]
Love the styling on the README, looks really good and I love all of the sections. With having the other PR's commits on this PR break anything if we merge?

**Comment by DesmondAldridge** [comment]
I believe since I did mistakenly branch off that branch, it's more or less two birds with one PR. lol

**Pull request merged**
